packages/hash/frontend/src/theme/components/navigation/MuiMenuThemeOptions.ts

```diff
@@ -0,0 +1,17 @@
+import { Components, Theme } from "@mui/material";
+
+export const MuiMenuThemeOptions: Components<Theme>["MuiMenu"] = {
+  defaultProps: {
+    elevation: 4,
+    autoFocus: false,
+  },
+  styleOverrides: {
+    list: {
+      paddingTop: "4px",
+      paddingBottom: "4px",
```

Comment:
```suggestion
p: 0.5,
```

In my opinion, if we're enforcing padding on menu items acc to the design, we should enforce it across both axes.

[The design system](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=831%3A67739) has a padding of `4px` applied across both axes.

Reply:
The reason I didn't enforce the 4px across the x-axis is because there are some scenarios where we have a Divider in the Menu and the horizontal padding should not apply to it. Instead I add a 4px horizontal margin to each `MenuItem` in `MuiMenuItemThemeOptions`

Reply:
Doesn't have to be changed, but I think the right approach would be to define the vertical and horizontal padding here, and override the horizontal padding in edge cases where they come up (such as the `Divider` component).

Reply:
> The problem with that approach is we'd have to either extra styling to the Divider component each time we use it or create a styled component for the Divider with negative margin.
I was suggesting to override the the styling of the menu list to set the horizontal styling to 0, so there wouldn't be the need for any negative margins. But I see your point that we will have to do this a lot.

What if we were to set horizontal margins in the `MuiMenuItemThemeOptions` instead? Then by default everything doesn't have horizontal margins, except the menu items which appears to be the most common use-case to my understanding.

Reply:
Ahh I see. The horizontal margin on MenuList is 0 by default. I currently set the horizontal margins in MuiMenuItemThemeOptions, [link](https://github.com/hashintel/hash/blob/bd74c7399775039599744fe3a5204c101f198077/packages/hash/frontend/src/theme/components/navigation/MuiMenuItemThemeOptions.ts#L85-L88). The horizontal margin can be removed by passing a `disableGutters` prop to the MenuItem